wire-ios/Wire-iOS/Sources/UserInterface/Conversation/Create/ConversationCreationController.swift

```diff
@@ -299,7 +299,9 @@ extension ConversationCreationController: AddParticipantsConversationCreationDel
                 enableReceipts: values.enableReceipts,
                 messageProtocol: values.encryptionProtocol == .proteus ? .proteus : .mls
             ) { [weak self] in
-                guard let self = self else { return }
+                guard let self = self else {
+                    assertionFailure("expect ConversationService not to be <nil>")
+                    return }
```

Comment:
💅 the indentation looks off. 
Is the message "expect ConversationService not to be <nil>" correct? I think it's about self (`ConversationCreationController`) here?

Reply:
fixed